pl_examples/domain_templates/reinforce_learn_Qnet.py

```diff
@@ -424,7 +420,7 @@ def main(args) -> None:
     torch.manual_seed(0)
     np.random.seed(0)
 
-    parser = argparse.ArgumentParser()
+    parser = argparse.ArgumentParser(add_help=False)
```

Comment:
Why did you set `add_help=False` ? It is always better to enable help.

Reply:
The code doesn't run with `add_help=True` ( #5382)

Since we supply our own help text for each argument, I think we can set this as `False`. With set to `False`, running `python pl_examples\domain_templates\reinforce_learn_Qnet.py --help` still displays the custom help options we provided.

Reply:
edit: here `add_help=False` is in the parent parser, but for the parser adding model specific args `add_help=True` by default. When it's set to `True` on both parsers, this conflicting option error occurs so one of them can be set to `False`